func aa(){
    print "";
}

func a(){
    print "a";
    aa(1);
}
func b(){
    print "b";
    a();
}

func c() {
    print "c";
    b();
}
try {
    c();
} 
except Exception as e
{
    print "Caught "+e.msg;
}
